ui-kit/RowCard/RowCard.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Divider, systemPropTypes } from 'ui-kit';
+import { Box } from 'ui-kit';
+// import Styled from 'ui-kit/Card/Card.styles';
+import Styled from './RowCard.styles';
+
+const RowCard = (props = {}) => {
+  return (
+    <Styled display="flex" {...props} minHeight={100}>
```

Comment:
Should `minHeight` go before `{...props}` here so it can be overridden?

Reply:
yeah good call!